git/git.go

```diff
@@ -615,6 +615,32 @@ func GitAndRootDirs() (string, string, error) {
 	return absGitDir, absRootDir, nil
 }
 
+func PathFromRoot(pattern string) ([]string, error) {
```

Comment:
> I considered making the function just return one path or error if there are more, but it seems to me the evolution of the lock commands will/should include supporting multiple files at a time

I think that it should be the caller's (in this case, the `commands` package, and particularly, `commands/commands_lock.go`) to break up multiple arguments into multiple calls. I think this could look something like:

```go
for _, lpath := range args {
        path, err := git.PathFromRoot(lpath)
        // ...
}
```

That being said, I think I still prefer my original suggestion of having both:

- `PathFromRoot(pattern string) (path string, err error)`
- `PathsFromRoot(pattern string) (paths []string, err error)`

Would you be opposed to going forward with this?